### PR TITLE
Fix a regression from adding negation support in #5625.

### PR DESCRIPTION
**Pull request by loganfsmyth**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | Y
| Major: Breaking Change?  | 
| Minor: New Feature?      | 
| Deprecations?            | 
| Spec Compliancy?         | 
| Tests Added/Pass?        | 
| Fixed Tickets            | 
| License                  | MIT
| Doc PR                   | 
| Dependency Changes       | 

`micromatch.filter()` does not work like I thought it did, woops.

### CODE REVIEW

File: packages/babel-core/test/config-chain.js [hunk 1]
```diff
@@ -31,7 +31,26 @@ describe("buildConfigChain", function () {
     process.env.NODE_ENV = oldNodeEnv;
   });
 
-  // TODO: Tests for ignore and only
+  describe("ignore/only", () => {
+    // TODO: More tests for ignore and only
```
Comment (xtuc) [inline packages/babel-core/test/config-chain.js]: We could create a beginner-friendly issue for this?

Comment (loganfsmyth) [inline packages/babel-core/test/config-chain.js]: Good call, will do.

Comment (xtuc) [inline packages/babel-core/test/config-chain.js]: I can take the time to create it, is that good for you?

Comment (loganfsmyth) [inline packages/babel-core/test/config-chain.js]: Beat you to it: https://github.com/babel/babel/issues/5645

**Pull request merged**